rockcraft.yaml

```diff
@@ -0,0 +1,46 @@
+name: mlflow-prometheus-exporter
+summary: MLflow Prometheus Exporter Image
+description: |
+  The Docker image for the Prometheus Python exporter for MLflow is a self-contained package 
+  that collects and exposes custom metrics from MLflow servers. It integrates seamlessly with 
+  MLflow deployments, leveraging the MLflow REST API to gather metrics such as registered 
+  models and deployments. By deploying this image as a containerized service, users can 
+  enhance observability and streamline monitoring and analysis workflows for MLflow using 
+  Prometheus and the Observability Stack.
+
+version: "v1.0.0_22.04"
```

Comment:
Let's have a version `v1.0.0_22.04_1`
I know we ARE "upstream" for this image, but we should be consistent. Plus. we might want to fix something without doing bump in v1.0.0.